pinot-controller/src/test/java/org/apache/pinot/controller/helix/core/rebalance/TableRebalancerClusterStatelessTest.java

```diff
@@ -66,13 +74,16 @@ public class TableRebalancerClusterStatelessTest extends ControllerTest {
   private static final String TIER_A_NAME = "tierA";
   private static final String TIER_B_NAME = "tierB";
   private static final String TIER_FIXED_NAME = "tierFixed";
+  private static final TableRebalanceObserver TABLE_REBALANCE_OBSERVER = mock(TableRebalanceObserver.class);
```

Comment:
this class was already testing a bunch of cases, so adding a mock here to check the observer seemed the best way to test.